packages/pwa-kit-react-sdk/src/ssr/universal/components/with-react-query/index.js

```diff
@@ -54,9 +54,9 @@ export const withReactQuery = (Wrapped, options = {}) => {
 
             return (
                 <QueryClientProvider client={this.props.locals.__queryClient}>
-                    <Hydrate state={preloadedState}>
+                    <HydrationBoundary state={preloadedState}>
```

Comment:
`Hydrate` is removed, and replaced by `HydrationBoundary`.

It's only a component name change, no props changes! :)